### ISSUE TITLE
Invalid resource attribute

### ISSUE THREAD

**Issue by httpdss** [aws-cloudformation/cfn-python-lint#1711]
*cfn-lint version: (`cfn-lint --version`)*
0.36.1
*Description of issue.*
running the command on a very simple cloudformation, give following errors
```
E3001 Invalid resource attribute AutoMinorVersionUpgrade for resource MySQLDBName
./rds/mysql_template.yaml:46:5

E3001 Invalid resource attribute BackupRetentionPeriod for resource MySQLDBName
./rds/mysql_template.yaml:47:5

E3001 Invalid resource attribute CopyTagsToSnapshot for resource MySQLDBName
./rds/mysql_template.yaml:48:5

E3001 Invalid resource attribute DBSecurityGroups for resource MySQLDBName
./rds/mysql_template.yaml:50:5

E3001 Invalid resource attribute Port for resource MySQLDBName
./rds/mysql_template.yaml:52:5

E3001 Invalid resource attribute PubliclyAccessible for resource MySQLDBName
./rds/mysql_template.yaml:53:5
```
updated def with -u but no luck

**Comment by kddejong** [comment]
Based on the properties in this message I'm guessing you are testing a resource of type `AWS::RDS::DBInstance` correct?

**Comment by PatMyron** [comment]
Resource type specific properties need to be indented underneath `Properties`:
```yaml
Resources:
  Resource:
    Type: AWS::RDS::DBInstance
    Properties:
      PubliclyAccessible: false
```
